README.rst

```diff
@@ -1,21 +1,28 @@
-*******************
-Nengo documentation
-*******************
+*********************
+About nengo.github.io
+*********************
 
 This repository is home to documentation
 of the whole Nengo ecosystem.
 
 How to contribute
 =================
 
-Documentation is written in
-`restructedText <http://docutils.sourceforge.net/rst.html>`_,
-built with `Sphinx <http://www.sphinx-doc.org/>`_
-automatically using `TravisCI <https://travis-ci.org/>`_
-and hosted with `Github pages <https://pages.github.com/>`_.
+This documentation is
+
+- hosted at `nengo/nengo.github.io <https://github.com/nengo/nengo.github.io>`_
+- written in `restructedText <http://docutils.sourceforge.net/rst.html>`_
+- built with `Sphinx <http://www.sphinx-doc.org/>`_
+- built automatically using `TravisCI <https://travis-ci.org/>`_
```

Comment:
nitpick: both lines say “built” 🤔 I'd be fine with that, but how about “deployed automatically using”?